pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -17,61 +17,83 @@
 package connections
```

Comment:
could changes to this package have been in a separate PR?

Reply:
I did not get the comment. Do you mean the earlier PR, where I added this package originally?

Reply:
I mean making these changes in a separate PR to avoid submitting a large PR all at once. It seems that these changes are not really about export, more about improving the connection tracker polling code?

Reply:
Yes, these are not related to IPFIX export, but the changes are required for the e2e test. Felt it is good to have e2e test in the same PR, as it is a layer of validation. If e2e test can be moved, I can move the changes.

Reply:
It's more for future work. Having smaller PRs is easier to review.

Reply:
Point taken. As I kept working on it, PR got bloated up and did not realize the size.  In hindsight, I should have had separate PRs for flow records pkg and exporter pkg. Maybe have the e2e test in a separate PR as well, as there is feature gate.